lib/components/SelectionControl/ToggleSwitch/toggle-switch.stories.tsx

```diff
@@ -0,0 +1,104 @@
+import { StoryObj, Meta } from "@storybook/react";
+import ToggleSwitch from ".";
+import "./toggle-switch.scss";
+import { fn } from "@storybook/test";
+
+const meta = {
+    title: "Components/Selection control/Toggle Switch",
+    component: ToggleSwitch,
+    parameters: { layout: "centered" },
+    tags: ["autodocs"],
+    args: {
+        defaultChecked: false,
+        disabled: false,
+        onChange: fn(),
+        onFocus: fn(),
+        onBlur: fn(),
+        className: "",
```

Comment:
Empty string is `falsy` value, so it can be considered same way as `undefined`. You can omit it here i guess.
Not important, you can ignore